File: java/program/AnggotaDPR.java
import java.util.ArrayList;
import java.util.Scanner;

public class AnggotaDPR {
    private int id;
    private String nama;
    private int umur;
    private String jabatan;
    private String namaBidang;
    private String namaPartai;

    public AnggotaDPR() {
        this.id = 0;
        this.nama = "";
        this.umur = 0;
        this.jabatan = "";
        this.namaBidang = "";
        this.namaPartai = "";
    }

    public AnggotaDPR(int id, String nama, int umur, String jabatan, String namaBidang, String namaPartai) {
        this.id = id;
        this.nama = nama;
        this.umur = umur;
        this.jabatan = jabatan;
        this.namaBidang = namaBidang;
        this.namaPartai = namaPartai;
    }

    public int getId() {
        return id;
    }

    public String getNama() {
        return nama;
    }

    public int getUmur() {
        return umur;
    }

    public String getJabatan() {
        return jabatan;
    }

    public String getNamaBidang() {
        return namaBidang;
    }

    public String getNamaPartai() {
        return namaPartai;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public void setUmur(int umur) {
        this.umur = umur;
    }

    public void setJabatan(String jabatan) {
        this.jabatan = jabatan;
    }

    public void setNamaBidang(String namaBidang) {
        this.namaBidang = namaBidang;
    }

    public void setNamaPartai(String namaPartai) {
        this.namaPartai = namaPartai;
    }

    public void showIdentitas(int idWidth, int namaWidth, int umurWidth, int jabatanWidth, int bidangWidth, int partaiWidth) {
        System.out.printf("%-" + idWidth + "d %-"
            + namaWidth + "s %-"
            + umurWidth + "d %-"
            + jabatanWidth + "s %-"
            + bidangWidth + "s %-"
            + partaiWidth + "s %n",
            id, nama, umur, jabatan, namaBidang, namaPartai);
    }
}

File: php/program/AnggotaDPR.php
<?php
class AnggotaDPR {
    private $id;
    private $nama;
    private $umur;
    private $jabatan;
    private $namaBidang;
    private $namaPartai;

    public function __construct($id, $nama, $umur, $jabatan, $namaBidang, $namaPartai) {
        $this->id = $id;
        $this->nama = $nama;
        $this->umur = $umur;
        $this->jabatan = $jabatan;
        $this->namaBidang = $namaBidang;
        $this->namaPartai = $namaPartai;
    }

    public function get_id() {
        return $this->id;
    }
    
    public function get_nama() {
        return $this->nama;
    }
    
    public function get_umur() {
        return $this->umur;
    }
    
    public function get_jabatan() {
        return $this->jabatan;
    }
    
    public function get_namaBidang() {
        return $this->namaBidang;
    }
    
    public function get_namaPartai() {
        return $this->namaPartai;
    }

    public function set_id($id) {
        $this->id = $id;
    }

    public function set_nama($nama) {
        $this->nama = $nama;
    }

    public function set_umur($umur) {
        $this->umur = $umur;
    }

    public function set_jabatan($jabatan) {
        $this->jabatan = $jabatan;
    }

    public function set_namaBidang($namaBidang) {
        $this->namaBidang = $namaBidang;
    }

    public function set_namaPartai($namaPartai) {
        $this->namaPartai = $namaPartai;
    }
    
    public function showIdentitas() {
        echo "<tr>
                <td>{$this->id}</td>
                <td>{$this->nama}</td>
                <td>{$this->umur}</td>
                <td>{$this->jabatan}</td>
                <td>{$this->namaBidang}</td>
                <td>{$this->namaPartai}</td>
              </tr>";
    }
}
?>
